src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -150,6 +163,35 @@ public static void main(String[] args) {
     System.exit(0);
   }
 
+  List<String> parseDepgraphRoots() {
+    List<String> result = new ArrayList<>();
+    for (String file : opts.depgraphFiles) {
+      try (FileReader reader = new FileReader(new File(file))) {
+        List<List> list = new Gson().fromJson(reader, new TypeToken<List<List>>(){}.getType());
```

Comment:
I agree we may want that, but I'm not sure we'll need anything beyond roots from the file (for example Laszlo says the provides listed in there may be different from the source closure sees, and we want the latter.)
I'll leave a todo for myself for now, I'd rather spend time fixing some bugs first.